package.json

```diff
@@ -2,10 +2,25 @@
   "name": "telemetry-hub",
```

Comment:
you should rename the project

Reply:
also the scylla-server name in its package.json is wrong and should be updated to scylla (that's my b)